### PR TITLE
Fix comments, formatting, exports in Accounting

### PR DESCRIPTION
**Pull request by paxthemax**
Added missing comments, fixed existing documentation comments, format fixes

### CODE REVIEW

File: pkg/accounting/accounting.go [hunk 1]
```diff
@@ -24,31 +24,35 @@ var (
...
-	lock            sync.Mutex // lock to be held during any accounting action for this peer
-	reservedBalance uint64     // amount currently reserved for active peer interaction
+	// Lock to be held during any accounting action for this peer.
+	lock sync.Mutex
+	// Amount currently reserved for active peer interaction
+	reservedBalance uint64
 }
 
-// Options for accounting
+// Options ar options provided to Accounting.
```
Comment (ralph-pichler) [inline pkg/accounting/accounting.go:55]: are

File: pkg/accounting/accounting.go [hunk 2]
```diff
@@ -215,16 +226,17 @@ func (a *Accounting) Credit(peer swarm.Address, price uint64) error {
...
 func (a *Accounting) settle(peer swarm.Address, balance *accountingPeer) error {
 	oldBalance, err := a.Balance(peer)
 	if err != nil {
 		return fmt.Errorf("failed to load balance: %w", err)
 	}
 
-	// don't do anything if there is no actual debt
-	// this might be the case if the peer owes us and the total reserve for a peer exceeds the payment treshold
+	// Don't do anything if there is no actual debt.
+	// This might be the case if the peer owes us and the total reserve	for a
```
Comment (ralph-pichler) [inline pkg/accounting/accounting.go:238]: empty space mid-sentence

File: pkg/accounting/accounting.go [hunk 3]
```diff
@@ -236,8 +248,9 @@ func (a *Accounting) settle(peer swarm.Address, balance *accountingPeer) error {
 	paymentAmount := uint64(-oldBalance)
 	nextBalance := 0
 
-	// try to save the next balance first
-	// otherwise we might pay and then not be able to save, thus paying again after restart
+	// Try to save the next balance first.
+	// Otherwise we might pay and then not be able to save, forcing us ti pay
```
Comment (ralph-pichler) [inline pkg/accounting/accounting.go:252]: to pay

File: pkg/accounting/accounting.go (hunk 1 above)
Comment (janos) [inline pkg/accounting/accounting.go:55]: ar -> are

**Review by janos — COMMENTED** [review]
This PR looks to me like a cosmetic one related to comments and unexporting functions.

I did not manage to find where overflows are fixed. The PR should be renamed.

File: pkg/accounting/accounting.go (hunk 1 above)
Comment (paxthemax) [reply pkg/accounting/accounting.go:55]: Fixed.

File: pkg/accounting/accounting.go (hunk 3 above)
Comment (paxthemax) [reply pkg/accounting/accounting.go:252]: Fixed.

File: pkg/accounting/accounting.go (hunk 1 above)
Comment (paxthemax) [reply pkg/accounting/accounting.go]: Fixed.

File: pkg/accounting/accounting.go (hunk 2 above)
Comment (paxthemax) [reply pkg/accounting/accounting.go]: Fixed.

**Comment by paxthemax** [comment]
@janos agreed, renamed PR.

**Review by janos — APPROVED** [review]
Thanks @paxthemax.

**Review by metacertain — APPROVED** [review]
Thanks!

**Pull request merged**